CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   - `reportGranules` - Receives 1 message per granule in an execution
   - `reportPdrs` - Receives 1 message per PDR
 
+- **CUMULUS-1470** ECS Service Autoscaling now uses the metric configured for the service rather than the `ActivityScheduleTime` metric.
```

Comment:
```suggestion
- **CUMULUS-1470** ECS Service Autoscaling now uses a configurable metric for the service rather than the `ActivityScheduleTime` metric.
```